src/components/Chat/MessagesBody/MessageSet/index.tsx

```diff
@@ -59,6 +59,10 @@ type MessageSetProps = {
 
 const MessageSet = ({ chats, layoutType, roomInfo }: MessageSetProps) => {
   const { oid: userOid } = useValueRecoilState("loginInfo") || {};
+  const readAts = useMemo(() => {
+    const readAts = roomInfo?.part?.map((user) => user?.readAt);
+    return readAts;
+  }, [chats, roomInfo]);
```

Comment:
연산에서 `chats`가 사용되지 않아 dependency array 에서 제외되어도 될 것 같아요.

```suggestion
  const readAts = useMemo(
    () => roomInfo?.part?.map((user) => user?.readAt) || [],
    [roomInfo]
  );
```